examples/webgl_buffergeometry_indexed.html

```diff
@@ -0,0 +1,206 @@
+				margin: 0px;
+				overflow: hidden;
+			}
+
+			#info {
+				position: absolute;
+				top: 0px; width: 100%;
+				padding: 5px;
+			}
+
+			a {
+				color: #0080ff;
+			}
+
+		</style>
+	</head>
+	<body>
+
+		<div id="container"></div>
+		<div id="info"><a href="http://threejs.org" target="_blank" rel="noopener">three.js</a> webgl - buffergeometry - indexed</div>
+
+		<script src="../build/three.js"></script>
+
+		<script src="js/Detector.js"></script>
+		<script src="js/libs/stats.min.js"></script>
+		<script src="./js/libs/dat.gui.min.js"></script>
+
+		<script>
+
+			if ( ! Detector.webgl ) Detector.addGetWebGLMessage();
+
+			var camera, scene, renderer, stats;
+
+			var mesh;
+
+			init();
+			animate();
+
+			function init() {
+
+				//
+
+				camera = new THREE.PerspectiveCamera( 27, window.innerWidth / window.innerHeight, 1, 3500 );
+				camera.position.z = 64;
+
+				scene = new THREE.Scene();
+				scene.background = new THREE.Color( 0x050505 );
+
+				//
+
+				var ambientLight = new THREE.AmbientLight( 0xcccccc );
+				scene.add( ambientLight );
+
+				var light1 = new THREE.DirectionalLight( 0xffffff, 0.5 );
+				light1.position.set( 1, 1, 1 );
+				scene.add( light1 );
+
+				var light2 = new THREE.DirectionalLight( 0xffffff, 1.5 );
+				light2.position.set( 0, -1, 0 );
+				scene.add( light2 );
+
+				//
+
+				var geometry = new THREE.BufferGeometry();
+
+				var indices = [];
+
+				var vertices = [];
+				var normals = [];
+				var colors = [];
+
+				var size = 20;
+				var segments = 10;
+
+				var halfSize = size / 2;
+				var segmentSize = size / segments;
+
+				// generate vertices, normals and color data for a simple grid geometry
+
+				for ( var i = 0; i <= segments; i ++ ) {
+
+					var y = ( i * segmentSize ) - halfSize;
+
+					for ( var j = 0; j <= segments; j ++ ) {
+
+						var x = ( j * segmentSize ) - halfSize;
+
+						vertices.push( x, - y, 0 );
+						normals.push( 0, 0, 1 );
+
+						var r = ( x / size ) + 0.5;
+						var g = ( y / size ) + 0.5;
+
+						colors.push( r, g, 1 );
+
+					}
+
+				}
+
+				// generate indices (data for element array buffer)
+
+				for ( var i = 0; i < segments; i ++ ) {
+
+					for ( var j = 0; j < segments; j ++ ) {
+
+						var a = i * ( segments + 1 ) + ( j + 1 );
+						var b = i * ( segments + 1 ) + j;
+						var c = ( i + 1 ) * ( segments + 1 ) + j;
+						var d = ( i + 1 ) * ( segments + 1 ) + ( j + 1 );
+
+						// generate two faces (triangles) per iteration
+
+						indices.push( a, b, d ); // face one
+						indices.push( b, c, d ); // face two
+
+					}
+
+				}
+
+				//
+
+				geometry.setIndex( indices );
+				geometry.addAttribute( 'position', new THREE.Float32BufferAttribute( vertices, 3 ) );
+				geometry.addAttribute( 'normal', new THREE.Float32BufferAttribute( normals, 3 ) );
+				geometry.addAttribute( 'color', new THREE.Float32BufferAttribute( colors, 3 ) );
+
+				var material = new THREE.MeshPhongMaterial( {
+					specular: 0xffffff, shininess: 250,
```

Comment:
If the specular reflectance of the material is `0xffffff`, and the diffuse reflectance is greater than zero, then more than 100% of the light is being reflected.  Set `specular` to `0x111111`. Study the hot-spot before and after.

Reply:
Okay done 👍

Reply:
BTW: I've just copied the material from an other example. We might want to review the lightings in the examples at some point...

Reply:
Right. Check the outline pass demo. :/